pkg/baseline/packageBaseline.1.1.json

```diff
@@ -204,9 +204,6 @@
     "System.Net.WebSockets.Client": {
       "BaselineVersion": "4.3.0"
     },
-    "System.Numerics.Vectors": {
```

Comment:
Do we still need this in the packageBaseline? Same for other baseline above?

Reply:
For the other baseline, do we still need all the `InboxOn` entries now? e.g. https://github.com/dotnet/corefx/pull/37301/files#diff-122916076db7087dbc454352fada61eeL3135

Reply:
Don't modify this file.  It represents history.

Reply:
Same for the regular packageBaseline?

Reply:
For that one you only need to remove the entry for 4.6.0 since that package won't ship.  Otherwise leave it the same.